sdk/advisor/Azure.ResourceManager.Advisor/api/Azure.ResourceManager.Advisor.net8.0.cs

```diff
@@ -210,56 +399,169 @@ protected MockableAdvisorSubscriptionResource() { }
+        public static Azure.ResourceManager.Advisor.AdvisorResiliencyReviewData AdvisorResiliencyReviewData(Azure.Core.ResourceIdentifier id = null, string name = null, Azure.Core.ResourceType resourceType = default(Azure.Core.ResourceType), Azure.ResourceManager.Models.SystemData systemData = null, string reviewName = null, string workloadName = null, Azure.ResourceManager.Advisor.Models.ResiliencyReviewStatus? reviewStatus = default(Azure.ResourceManager.Advisor.Models.ResiliencyReviewStatus?), int? recommendationsCount = default(int?), string publishedAt = null, string updatedAt = null) { throw null; }
+        public static Azure.ResourceManager.Advisor.Models.AdvisorScoreEntityContent AdvisorScoreEntityContent(string date = null, float? score = default(float?), float? consumptionUnits = default(float?), float? impactedResourceCount = default(float?), float? potentialScoreIncrease = default(float?), float? categoryCount = default(float?)) { throw null; }
+        public static Azure.ResourceManager.Advisor.AdvisorScoreEntityData AdvisorScoreEntityData(Azure.Core.ResourceIdentifier id = null, string name = null, Azure.Core.ResourceType resourceType = default(Azure.Core.ResourceType), Azure.ResourceManager.Models.SystemData systemData = null, Azure.ResourceManager.Advisor.Models.AdvisorScoreEntityContent lastRefreshedScore = null, System.Collections.Generic.IEnumerable<Azure.ResourceManager.Advisor.Models.TimeSeriesEntity> timeSeries = null) { throw null; }
+        public static Azure.ResourceManager.Advisor.Models.AdvisorTriageResourceData AdvisorTriageResourceData(Azure.Core.ResourceIdentifier id = null, string name = null, Azure.Core.ResourceType resourceType = default(Azure.Core.ResourceType), Azure.ResourceManager.Models.SystemData systemData = null, string reviewId = null, string recommendationId = null, string subscriptionId = null, string resourceGroup = null, Azure.Core.ResourceIdentifier resourceId = null, string resourceName = null) { throw null; }
+        public static Azure.ResourceManager.Advisor.Models.AdvisorWorkloadResult AdvisorWorkloadResult(string id = null, string name = null, string subscriptionId = null, string subscriptionName = null) { throw null; }
+        public static Azure.ResourceManager.Advisor.Models.AssessmentTypeResult AssessmentTypeResult(string id = null, string title = null, string description = null, string locale = null, string version = null) { throw null; }
```

Comment:
From the usage of these two models I think Result is redundant.
Could we rename them to `AdvisorWorkload` and `AdvisorAssessmentType`